cpp/cmake/thirdparty/CUSPATIAL_GetCUDF.cmake

```diff
@@ -26,15 +38,15 @@ function(find_and_configure_cudf VERSION)
         set(MAJOR_AND_MINOR "${VERSION}")
     endif()
 
-    CPMFindPackage(NAME        cudf
-        VERSION                ${VERSION}
-        GIT_REPOSITORY         https://github.com/rapidsai/cudf.git
-        GIT_TAG                branch-${MAJOR_AND_MINOR}
-        GIT_SHALLOW            TRUE
-        SOURCE_SUBDIR          cpp
-        OPTIONS                "BUILD_TESTS OFF"
-                               "BUILD_BENCHMARKS OFF"
-        FIND_PACKAGE_ARGUMENTS "COMPONENTS testing")
+    rapids_cpm_find(
+      cudf ${VERSION}
+      CPM_ARGS
+      GIT_REPOSITORY https://github.com/rapidsai/cudf.git
+      GIT_TAG branch-${MAJOR_AND_MINOR}
+      GIT_SHALLOW TRUE
+      OPTIONS "BUILD_TESTS OFF" "BUILD_BENCHMARKS OFF"
+      FIND_PACKAGE_ARGUMENTS "COMPONENTS testing"
+    )
 
     # Make sure consumers of cuspatial can see cudf::cudf
     fix_cmake_global_defaults(cudf::cudf)
```

Comment:
We can fully remove `fix_cmake_global_defaults` and instead specify `cudf::cudf` as a `GLOBAL_TARGET` in the `rapids_cpm_find` call.

https://docs.rapids.ai/api/rapids-cmake/stable/command/rapids_cpm_find.html